argocon-demos/2023-migrating-cicd-from-jenkins-to-argo/README.md

```diff
@@ -3,9 +3,9 @@
 [![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io?utm_campaign=talk-demos)
 
 ## The talk
-<!-- The talk [can be found here](https://youtu.be/g8TCgnrM648).
+<!-- The talk recording [can be found here](https://youtu.be/g8TCgnrM648).-->
 
-The slide deck for this talk can be found [here](assets/slide-deck.pdf). -->
+The slide deck for this talk can be found [here](slides/Migrating-CI_CD-Jenkins-Argo-ARCOGON_NA2023.pdf). 
```

Comment:
this doesn't seem to exist?

Reply:
Odd, I thought I added it to the branch and PR

Reply:
I take it back. Github helpfuly showed me a single commit rather than the whole branch. sorry